app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -529,7 +539,8 @@ bool OpenXRInputSource::GetHandTrackingInfo(XrTime predictedDisplayTime, XrSpace
     XrHandJointLocationsEXT jointLocations { XR_TYPE_HAND_JOINT_LOCATIONS_EXT };
     jointLocations.jointCount = XR_HAND_JOINT_COUNT_EXT;
     jointLocations.jointLocations = mHandJoints.data();
-    mGestureManager->populateNextStructureIfNeeded(jointLocations);
+    if (mGestureManager)
```

Comment:
Is valid to have a nullptr for the  mGestureManager ?

Reply:
Now yes, because devices supporting the hand interaction profile do not use/need a gesture manager